### PR TITLE
Fix DNS wildcard support for EasyDNS

### PR DESCRIPTION
**Pull request by WarrenSchultz**
DNS wildcard is currently broken for EasyDNS. This patch fixes that issue.

### CODE REVIEW

**Comment by pfSense-git-updates** [comment]
Before this pull request can be accepted you must first sign a CLA as described at https://www.pfsense.org/about-pfsense/#cla. Please read for more details.

**Comment by WarrenSchultz** [comment]
CLA in place.

File: src/etc/inc/dyndns.class [hunk 1]
```diff
@@ -450,6 +450,8 @@
 					break;
 				case 'easydns':
 					$needsIP = TRUE;
+					if (isset($this->_dnsWildcard))
+						$this->_dnsWildcard = "ON";
```
Comment (rbgarga) [inline src/etc/inc/dyndns.class]: Like other places in the code you should check if it's != "OFF" before assume it should be "ON"

**Comment by sbeaver-netgate** [comment]
No action on requested changes.

**Pull request closed without merging**